frontend/app/views/help/syntax.html.haml

```diff
@@ -221,4 +232,4 @@
       = search_help "sort:new", "sort results by most recent first, counting only regular sets for sake of order (other printings will go at end of results)"
       = search_help "sort:newall", "sort results by most recent first, counting all printings"
       = search_help "sort:ord", "sort results by oldest first, counting only regular sets for sake of order"
-      = search_help "sort:ordall", "sort results by oldest first, counting all printings"
+      = search_help "sort:ordall", "sort results by oldest first, counting all printings"
```

Comment:
It seems that your editor deletes final newline here. This is going to cause unnecessary conflicts in git patches.